web/source/kmwosk.js

```diff
@@ -792,14 +807,12 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
         if(typeof Lkc.Lcode == 'undefined')
         {
           Lkc.Lcode = osk.getVKDictionaryCode(keyName);// Updated for Build 347
-          if(!Lkc.Lcode)
+          if(!Lkc.Lcode) 
           {
-            // Key code will be Unicode value for U_xxxx keys
-            if(keyName.substr(0,2) == 'U_')
-            {
-              var tUnicode=parseInt(keyName.substr(2),16);
-              if(!isNaN(tUnicode)) Lkc.Lcode=tUnicode;
-            }
+            // Special case for U_xxxx keys. This vk code will never be used
+            // in a keyboard, so we use this to ensure that keystroke processing
+            // occurs for the key.
+            Lkc.Lcode = 1; 
```

Comment:
Are we saying that this assigned code never gets utilized, but merely ensures we successfully pass any later `if(!Lkc.Lcode)` tests?